iteround/core.py

```diff
@@ -65,7 +65,7 @@ def saferound(iterable, places, strategy=DIFFERENCE, rounder=round):
 
     # calculate original sum, rounded,  then rounded local sum.
     local = [Number(i, value) for i, value in enumerate(values)]
-    orig_sum = _sumnum(local, places, rounder)
+    orig_sum = _sumnum(local, places, rounder) if topline == -1 else topline
```

Comment:
Style semantics, but if you change `topline`'s default value to `None`, then we can use the following pythonic expression.

`orig_sum = topline or _sumnum(local, places, rounder)`